reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+- Intentionally creating space at meetings for new contributors to get involved
+  and/or a dedicated triage meeting that provide an overview of current 
+  priorities
+- Programs such as [Meet Our Contributors], an Office hours like space where
+  people can ask questions related to contributing.
+- Contributor events such as Contributor Summits, or Maintainer focused sessions
+  at KubeCon.
+- One-on-One sessions for dedicated contributors that have demonstrated a vested
+  interest in contributing and climbing the contributor ladder.
+- Group study groups for reviewers, approvers, and Chairs
+
+
+[Meet Our Contributors]: https://www.kubernetes.dev/events/meet-our-contributors/
+
+
+#### Areas of research:
+
+- specific outreach to new contributors from backgrounds that are
+  underrepresented in the community, such as BIPOC contributors
+- GitHub automation that will suggest contributors to promote for more
+  visibility to OWNERs 
+- ways to encourage regular review of who is actively reviewing in subprojects
+- scaling the [group contributor ladder program]
+
+
+[group contributor ladder program]: https://git.k8s.io/community/mentoring/programs
+
+
+## /help-wanted
+
+The community groups report that they need to grow code and non-code contributors
+into OWNERs files to help maintain the project. 
+
+Below is list of specific contribution needs, special projects, roles available,
+and more. Building trust is key and we need folks to stick around. There are
+other ways of contributing outside of commits and you'll see those in the
+[Other Types of Upstream Contributions] section.
+
+[Other types of upstream contributions]: #Other-types-of-upstream-contributions
+
+**SIG API Machinery**:
+- Performing triage (go to a triage meeting and you'll see it first hand)
+- Contributors to the Client Libraries like client-go, python-client
+
+**SIG Architecture**:
+- Site Reliability Engineers to review KEPs, Production Readiness Reviews,
+  and API Reviewers
+- Contributors to help curate a mentoring program for people to work across SIGs
+
+**SIG Auth**:
+- Audit logging and testing contributors
+- KMS-Plugin contributors
+
+**SIG Autoscaling**:
+- Creating and running of a triage program
+
+**SIG CLI**:
+- A Product/Feature Manager
+
+**SIG Cloud Provider**:
+- more contributors from every cloud to form teams for triage and support, 
+- folks from clouds to help run the cloud provider extraction working group:
+  kubernetes/kubernetes "cluster" directory and resolving how we properly test
+  kubernetes/kubernetes in the absence of a cloud provider
+
+**SIG Cluster Lifecycle**:
+- Etcdadm
+- Cluster-addons
+- Kubeadm contributors 
+
+**SIG Contributor Experience**: 
+- full-time (or part-time) community managers 
+- automation engineers: zoom to youtube automation, github automation, slack
+  infrastructure, and more
+- recognition programs for contributors
+- and folks to help create/facilitate contributor ladder mentoring programs
```

Comment:
```suggestion
- Full-time (or part-time) community managers 
- Automation engineers: zoom to youtube automation, github automation, slack
  infrastructure, and more
- Recognition programs for contributors
- People to create/facilitate contributor ladder mentoring programs
```

"Recognition programs for contributors" -> I get that you want these, but what kind of person do you want? :)